tests/clar_libgit2.c

```diff
@@ -319,6 +319,35 @@ const char* cl_git_path_url(const char *path)
 	return url;
 }
 
+const char *cl_git_sandbox_path(int is_dir, const char *path, ...)
+{
+	static char _temp[4096];
+	git_buf buf = GIT_BUF_INIT;
+	va_list arg;
+
+	cl_git_pass(git_buf_sets(&buf, clar_sandbox_path()));
+	if (path) {
```

Comment:
This check is kind of superfluous. The signature should rather be `const char *cl_git_sandbox_path(int is_dir, ...)`. Then you could avoid this additional check and just enter right into the `while (va_arg(...) != NULL)` loop.

Reply:
True. `is_dir` was added afterward, hence the path test. Fixed.